x/concentrated-liquidity/concentrated-pool/pool.go

```diff
@@ -0,0 +1,101 @@
+package concentrated_pool
```

Comment:
let's name this either `concentratedpool` or `concentrated` please

Importing packages with underscore isn't convenient

Reply:
Why do you think they are inconvenient?

Reply:
When we import this package, we need to refer to it as `concentrated_pool` .

E.g.: `concentrated_pool.RegisterInterfaces(registry)`

Which is inconsistent with style patterns in Go